plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -400,61 +390,59 @@ decodeBuiltin = dBEBits8 builtinTagWidth
-              Sha2_256                 -> 13
-              Sha3_256                 -> 14
-              VerifySignature          -> 15
-              EqualsByteString         -> 16
-              QuotientInteger          -> 17
-              ModInteger               -> 18
-              LessThanByteString       -> 19
-              GreaterThanByteString    -> 20
-              IfThenElse               -> 21
-              CharToString             -> 22
-              Append                   -> 23
-              Trace                    -> 24
-              Nop1                     -> 25
-              Nop2                     -> 26
-              Nop3                     -> 27
-              EqualsString             -> 28
-              EncodeUtf8               -> 29
-              DecodeUtf8               -> 30
-              FstPair                  -> 31
-              SndPair                  -> 32
-              NullList                 -> 33
-              HeadList                 -> 34
-              TailList                 -> 35
-              ConstrData               -> 36
-              MapData                  -> 37
-              ListData                 -> 38
-              IData                    -> 39
-              BData                    -> 40
-              UnConstrData             -> 41
-              UnMapData                -> 42
-              UnListData               -> 43
-              UnIData                  -> 44
-              UnBData                  -> 45
-              EqualsData               -> 46
-              ChooseData               -> 47
-              ChooseUnit               -> 48
-              MkPairData               -> 49
-              MkNilData                -> 50
-              MkNilPairData            -> 51
-              MkCons                   -> 52
-              ChooseList               -> 53
-              Blake2b_256              -> 54
+              AddInteger            -> 0
+              SubtractInteger       -> 1
+              MultiplyInteger       -> 2
+              DivideInteger         -> 3
+              RemainderInteger      -> 4
+              LessThanInteger       -> 5
+              LessThanEqualsInteger -> 6
+              EqualsInteger         -> 9
+              Concatenate           -> 10
+              TakeByteString        -> 11
+              DropByteString        -> 12
+              Sha2_256              -> 13
+              Sha3_256              -> 14
+              VerifySignature       -> 15
+              EqualsByteString      -> 16
+              QuotientInteger       -> 17
+              ModInteger            -> 18
+              LessThanByteString    -> 19
+              GreaterThanByteString -> 20
+              IfThenElse            -> 21
+              CharToString          -> 22
+              Append                -> 23
+              Trace                 -> 24
+              Nop1                  -> 25
```

Comment:
If you're changing this anyway, could you put the nops at the end?  I think I can get rid of them by just building a completely new builtins object that contains only them for use in the benchmarking process, and if and when I do that it would be good if the encoding didn't get disturbed any more.

Reply:
I'm planning to do a reordering of the builtins soonish. That's why I just left a hole this time, rather than doing anything cleverer.